spec/models/value_spec.rb

```diff
@@ -7,6 +7,52 @@
     it { is_expected.to be_a Hash }
   end
 
+  describe "##normalise_key (class method)" do
+    subject { described_class.normalise_key(key) }
+
+    context "when the key contains unsupported characters" do
+      let(:key) { "this-is!a{test}_key#123£567" }
+
+      it { is_expected.to be(:this_is_a_test__key_123_567) }
+
+      it { is_expected.to be_a Symbol }
+    end
+  end
+
+  describe "##get (class method)" do
+    subject { described_class.get(key) }
+
+    context "when the key contains hyphens" do
+      let(:key) { "dates-example-closing" }
+
+      it { is_expected.to eql("31/12/2024") }
```

Comment:
where does this date come from, and will the test break next year?

Reply:
the key is an example in an earlier PR and is used as part of the documentation, so shouldn't necessarily be updated next year.